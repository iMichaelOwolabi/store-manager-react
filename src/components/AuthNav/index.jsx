import React, { Fragment } from 'react';
import { ToastContainer } from 'react-toastify';
import { Link } from 'react-router-dom';
import { Icon } from 'semantic-ui-react';
import 'react-toastify/dist/ReactToastify.css';
import '../NavBar/NavBar.scss';

export default function AuthNav() {
  return (
    <Fragment>
      <header>
        <ToastContainer autoClose={5000} />
        <div className="container">
          <div id="logo">
            <h1>Store Manager</h1>
          </div>
          <nav>
            <ul>
              <Link to="/dashboard" className="navbar">Dashboard</Link>
              <Link to="/products" className="navbar">Products</Link>
              <Link to="/sales" className="navbar">Sales</Link>
              <Link to="/attendants" className="navbar">Attendants</Link>
              <Link to="/" className="navbar">Logout</Link>
              <Icon name="shop" className="cart" size="large" />
            </ul>
          </nav>
        </div>
      </header>
    </Fragment>
  );
}
