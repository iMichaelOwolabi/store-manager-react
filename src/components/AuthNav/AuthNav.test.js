import React from 'react';
import { shallow } from 'enzyme';
import AuthNav from '.';

describe('<Narbar Component />', () => {
  it('should', () => {
    const wrapper = shallow(<AuthNav />);
    expect(wrapper).toMatchSnapshot();
  });
});
