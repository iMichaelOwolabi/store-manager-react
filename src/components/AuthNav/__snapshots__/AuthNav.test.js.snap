// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Narbar Component /> should 1`] = `
<Fragment>
  <header>
    <ToastContainer
      autoClose={5000}
      bodyClassName={null}
      className={null}
      closeButton={
        <CloseButton
          ariaLabel="close"
        />
      }
      closeOnClick={true}
      draggable={true}
      draggablePercent={80}
      hideProgressBar={false}
      newestOnTop={false}
      pauseOnFocusLoss={true}
      pauseOnHover={true}
      position="top-right"
      progressClassName={null}
      progressStyle={null}
      rtl={false}
      style={null}
      toastClassName={null}
      transition={[Function]}
    />
    <div
      className="container"
    >
      <div
        id="logo"
      >
        <h1>
          Store Manager
        </h1>
      </div>
      <nav>
        <ul>
          <Link
            className="navbar"
            replace={false}
            to="/dashboard"
          >
            Dashboard
          </Link>
          <Link
            className="navbar"
            replace={false}
            to="/products"
          >
            Products
          </Link>
          <Link
            className="navbar"
            replace={false}
            to="/sales"
          >
            Sales
          </Link>
          <Link
            className="navbar"
            replace={false}
            to="/attendants"
          >
            Attendants
          </Link>
          <Link
            className="navbar"
            replace={false}
            to="/"
          >
            Logout
          </Link>
          <Link
            replace={false}
            to="/cart"
          >
            <Icon
              as="i"
              className="cart"
              name="shop"
              size="large"
            />
          </Link>
        </ul>
      </nav>
    </div>
  </header>
</Fragment>
`;
